### ISSUE TITLE
ARI Notifications not triggering

### ISSUE THREAD

**Issue by rutwikGit** [apaleo/api#42]
## Question

We have recently started working on integration of CRS with Apaleo and using the APIs we have been able to get property details and also create subscription request.
However even after successful creation of subscription ARI notifications are not being pushed to the requested URL and even force full refresh request doesn't trigger anything

## Background

Property Code: SHRTSTEU1
Subcription Id : 46D19GLY
Environment: Test (https://distribution.apaleo.io/)

**Comment by pgrm** [comment]
@rutwikGit we are sending you constant notifications, every 5 minutes, but your server never responds with a success status code (2xx)

Please keep in mind, the push is an HTTP POST method. You can find more details in the guide linked from the swagger specification: https://apaleo.com/channel-integration-guide/

**Comment by rutwikGit** [comment]
@pgrm I am not able to see any entry in our web-server log showing that the data is being posted to it for the URL registered agains the 46D19GLY subscription id

I tired doing a test post to the same URL using Postman from my PC and I can see that entry in web-server logs and also the ARI has been successfully posted

Can you please let us know what URL the data is being posting from Apaleo and a sample message being posted and the response received back along with Http StatusCode so that I can troubleshoot the same from my side also.

**Comment by pgrm** [comment]
@rutwikGit since you registered a token in the URL, please send an email to support@apaleo.com referencing this issue and I'll reply to it with the full URL.

This is the body being sent:

[request-body.zip](https://github.com/apaleo/api/files/2607077/request-body.zip)

**Comment by rutwikGit** [comment]
I've sent email from my company email ID, please check

**Comment by ozonni** [comment]
There seems to be an issue with SSL certificate that is used for your endpoint. Please make sure that it's possible to establish secure connection to your endpoint and it can receive https requests.

**Comment by rutwikGit** [comment]
@ozonni That is strange, because I am able to access the same URL form browser without any SSL error

**Comment by ozonni** [comment]
try to send POST request with the payload using curl or Postman or any other tool and see if it goes through.

**Comment by ozonni** [comment]
Seems like your certificate is signed with an intermediate CA certificate from one of the Go Daddy CA's and your server is misconfigured in the regard that it doesn't include that intermediate CA certificate as a CA chain it's response. Check out [What is an intermediate certificate?](https://www.godaddy.com/help/what-is-an-intermediate-certificate-868). That's the only thing i could think of.

**Comment by ozonni** [comment]
You can also do the [SSL Test](https://www.ssllabs.com/ssltest/analyze.html) and see that the certificate chain is incomplete.

**Comment by rutwikGit** [comment]
As per suggestion from our system admin I've now changed the domain name in the subscription URL and it seems to be working fine now.
Can you please confirm that you are getting success back from us now?

**Comment by ozonni** [comment]
Yes, we see that you have received ARI notifications and we got successful response back.
